index.html

```diff
@@ -2,26 +2,25 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title>FOSSASIA 2017 Singapore | Asia's Open Technology Event, Open Hardware, Free Software, Open Design, Open Data</title>
+        <title>FOSSASIA 2016 Singapore | Asia's Open Technology Event, Open Hardware, Free Software, Open Design, Open Data</title>
```

Comment:
@jig08 I have cloned your project using this link https://github.com/fossasia/2017.fossasia.org.git and the code I got on my PC is for 2016. This is the problem.